src/control/lib/control/pool_property.go

```diff
@@ -116,6 +123,50 @@ func PoolProperties() PoolPropertyMap {
 				jsonNumeric: true,
 			},
 		},
+		"ec_pda": {
+			Property: PoolProperty{
+				Number:      drpc.PoolPropertyECPda,
+				Description: "Performance domain affinity level of EC",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					ecpdaErr := errors.Errorf("invalid ec_pda value %s", s)
+					pdaPct, err := strconv.ParseUint(s, 10, 64)
```

Comment:
nit: `pdaPct` looks like a copy-paste error. Probably a better name is `pdaLvl` or something like that.